src/style.css

```diff
@@ -0,0 +1,134 @@
+body {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.stars--0 {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+}
+
+.stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  margin: 0 2px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+}
+
+.stars--2 {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+}
+
+.stars--3 {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+}
+
+.stars--4 {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+}
+
+.stars--5 {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+}
```

Comment:
add common class .star to html and style it, do not repeat the same code many times